daemon/stats.go

```diff
@@ -10,14 +10,18 @@ import (
 	"github.com/opencontainers/runc/libcontainer"
 )
 
+// ContainerStatsConfig holds information for configuring the runtime
+// behavior of a daemon.ContainerStats() call.
```

Comment:
Maybe this struct could be removed. The three items could just be arguments to ContainerStats.